controller/controller.go

```diff
@@ -1483,31 +1483,17 @@ func (ctrl *ProvisionController) provisionClaimOperation(ctx context.Context, cl
 		return ProvisioningFinished, errStopProvision
 	}
 
-	var selectedNode *v1.Node
+	var selectedNodeName string
 	// Get SelectedNode
 	if nodeName, ok := getString(claim.Annotations, annSelectedNode, annAlphaSelectedNode); ok {
-		if ctrl.nodeLister != nil {
```

Comment:
Do we still need the nodeLister for anything else? Can we remove it?

Reply:
We can remove it